### ISSUE TITLE
Atom unexpected token error when loading firacode from a git clone

### ISSUE THREAD

**Issue by TechieAndroid** [tonsky/FiraCode#1181]
```
Invalid or unexpected token
Hide Stack Trace
/home/recompiler/.atom/packages/FiraCode/distr/fira_code.css:1
(function (exports, require, module, __filename, __dirname, process, global, Buffer) { return function (exports, require, module, __filename, __dirname) { @font-face {
                                                                                                                                                           ^
SyntaxError: Invalid or unexpected token
    at new Script (vm.js:83:7)
    at NativeCompileCache.runInThisContext (/usr/lib/atom/src/native-compile-cache.js:43:20)
    at Module._compile (/usr/lib/atom/src/native-compile-cache.js:100:36)
    at Object.value [as .js] (/usr/lib/atom/src/compile-cache.js:255:23)
    at Module.load (internal/modules/cjs/loader.js:645:32)
    at Function.Module._load (internal/modules/cjs/loader.js:560:12)
    at Module.require (internal/modules/cjs/loader.js:685:19)
    at require (/usr/lib/atom/src/native-compile-cache.js:68:27)
    at Package.requireMainModule (/usr/lib/atom/src/package.js:907:27)
    at /usr/lib/atom/src/package.js:140:16
    at Package.measure (/usr/lib/atom/src/package.js:81:19)
    at Package.load (/usr/lib/atom/src/package.js:126:10)
    at PackageManager.loadAvailablePackage (/usr/lib/atom/src/package-manager.js:721:10)
    at /usr/lib/atom/src/package-manager.js:617:14
    at Config.transact (/usr/lib/atom/src/config.js:892:14)
    at PackageManager.loadPackages (/usr/lib/atom/src/package-manager.js:615:17)
    at /usr/lib/atom/src/atom-environment.js:949:21
    at async Promise.all (index 0)
    at AtomEnvironment.startEditorWindow (/usr/lib/atom/src/atom-environment.js:1044:20)
```

**Comment by tonsky** [comment]
Sorry, what exactly are you trying to do?

**Comment by TechieAndroid** [comment]
Sorry for not explaining it in detail. I'm trying to get firacode along with the ligatures to work inside atom.

So far I've tried several packages inside of atom to get it working but to no avail, I've even tried to set up a stylesheet and use the font selector but neither one of those worked. And that's what lead me to this error, I git cloned this repo into .atom/packages/ and reloaded atom and I got the error in my initial comment.
